protractor.conf.js

```diff
@@ -1,7 +1,10 @@
 var outputDir = 'test-result';
 var config = {
   capabilities: {
-    browserName: 'firefox'
+    browserName: 'chrome',
+    chromeOptions: {
+      w3c: false
```

Comment:
Switched to chrome with this config so that the right click action in the browser test can work.